features/conversation-list/conversation-list-item/conversation-list-item-dm.tsx

```diff
@@ -73,15 +78,17 @@ export const ConversationListItemDm = memo(function ConversationListItemDm({
 
   const renderRightActions = useCallback(
     (args: ISwipeableRenderActionsArgs) => {
-      return <ToggleUnreadSwipeableAction {...args} topic={topic} />;
+      return (
+        <ToggleUnreadSwipeableAction {...args} topic={conversationTopic} />
+      );
     },
-    [topic]
+    [conversationTopic]
   );
 
-  const deleteDm = useDeleteDm(conversation);
+  const deleteDm = useDeleteDm(conversation as DmWithCodecsType);
```

Comment:
again, not as part of this but we should start deriving these hooks from more primitive values (ids for example) as a best practice